src/views/SingleTableTournament/SingleTablePage.vue

```diff
@@ -8,38 +8,59 @@
     <ion-header>
       <ion-toolbar>
         <ion-buttons slot="start">
-          <ion-back-button default-href="/home" routerDirection="backward"></ion-back-button>
+          <ion-back-button text="" :icon="arrowBackOutline" default-href="/home" routerDirection="backward"></ion-back-button>
         </ion-buttons>
-        <ion-title>Nieuw tournament</ion-title>
+        <ion-title>Nieuw toernooi</ion-title>
       </ion-toolbar>
     </ion-header>
 
     <ion-content>
-      <page-title title="1 Table (2 VS 2)"></page-title>
-      <h2>Paar Noord / Zuid</h2>
-      <ion-item>
-        <ion-label>N: </ion-label>
-        <ion-input id="northName" placeholder="Naam van Noord" v-model="northInput"></ion-input>
-        <ion-label>Z: </ion-label>
-        <ion-input id="southName" placeholder="Naam van Zuid" v-model="southInput"></ion-input>
-      </ion-item>
-      <h2>Paar Oost / West</h2>
-      <ion-item>
-        <ion-label>O: </ion-label>
-        <ion-input id="eastName" placeholder="Naam van Oost" v-model="eastInput"></ion-input>
-        <ion-label>W: </ion-label>
-        <ion-input id="westName" placeholder="Naam van West" v-model="westInput"></ion-input>
-      </ion-item>
-      <h2>Scoring Principe</h2>
-      <ion-item>
-        <ion-label>Scoring Principe: </ion-label>
-        <ion-select interface="popover" v-model="principle" id="scoringPrinciple" placeholder="Selecteer scoring principe">
-          <ion-select-option value="theoretical">Theoretisch optimum</ion-select-option>
-        </ion-select>
-      </ion-item>
-      <ion-button :disabled="!allFieldsFilledIn" @click="() => gotoScanBoard($router)" router-direction="forward">
-        Start Spel
-      </ion-button>
+
+      <div class="h-align"> 
+        <div class="orange-box">
+          <h2>Voer de info in van het toernooi: </h2>
+        </div>
+      </div>
```

Comment:
Misschien beter om hier gewoon te zeggen dat alleen de namen zijn?
```suggestion
      <div class="h-align"> 
        <div class="orange-box">
          <h2>Voer de namen van de spelers in:</h2>
        </div>
      </div>
```